frontend/src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+import { API_BASE_URL } from "@/config";
+
+import { Footer as IFooter, Logo as ILogo } from "@types/Footer";
+
+export const Footer: React.FC<IFooter> = ({disclaimer, logos, privacy}) => {
+    return (
+        <div className="aha__footer">
+            <div className="disclaimer" dangerouslySetInnerHTML={{
+                __html: disclaimer,
```

Comment:
Perhaps this should be fixed in a separate issue, but I tested the markdown in the disclaimer and the privacy statement and it all looks a bit weird. I think some styles & classes are conflicting with each other, resulting in texts being centered, max-width's being applied, but also the default markdown/prose styling.

I think it would already help a lot if we just apply a `max-width: 700px` and `margin: 0 auto;` on the containers so that everything gets centered in a consistent way.

<img width="1341" alt="image" src="https://github.com/Amsterdam-Music-Lab/MUSCLE/assets/8208970/48b84d94-653a-456f-b64e-5821431ef658">